examples/lines_bars_and_markers/cohere_demo.py

```diff
@@ -0,0 +1,38 @@
+"""
+=====================================
+Plotting the coherence of two signals
+=====================================
+
+An example showing how to plot the coherence of two signals.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Fixing random state for reproducibility
+np.random.seed(19680801)
+
+dt = 0.01
+t = np.arange(0, 30, dt)
+nse1 = np.random.randn(len(t))                 # white noise 1
+nse2 = np.random.randn(len(t))                 # white noise 2
+r = np.exp(-t / 0.05)
+
+cnse1 = np.convolve(nse1, r, mode='same') * dt   # colored noise 1
+cnse2 = np.convolve(nse2, r, mode='same') * dt   # colored noise 2
```

Comment:
I'm not exactly sure what this "colored noise" means, but something is wrong here. If you don't limit the x-axis, you can see that past 15s, the noise is nearly zero. It works better if you use full mode and truncate the result:
```python
cnse1 = np.convolve(nse1, r)[:len(t)] * dt   # colored noise 1
cnse2 = np.convolve(nse2, r)[:len(t)] * dt   # colored noise 2 
```